### PR TITLE
Fixed currency fetch logic

### PR DESCRIPTION
**Pull request by BoyanPavlov03**

### CODE REVIEW

File: FinancialTracker/Helpers/Currency.swift [hunk 1]
```diff
@@ -101,3 +40,51 @@ extension Currency {
         rate = try values.decodeIfPresent(Double.self, forKey: .rate) ?? 0
     }
 }
+
+func getCurrencies(completionHandler: @escaping (String?, [Currency]?) -> Void) {
+    var allCurrencies: [Currency] = []
```
Comment (3257) [inline FinancialTracker/Helpers/Currency.swift]: Seems like this property is not needed.

File: FinancialTracker/Helpers/Currency.swift [hunk 2]
```diff
@@ -101,3 +40,51 @@ extension Currency {
         rate = try values.decodeIfPresent(Double.self, forKey: .rate) ?? 0
     }
 }
+
+func getCurrencies(completionHandler: @escaping (String?, [Currency]?) -> Void) {
```
Comment (3257) [inline FinancialTracker/Helpers/Currency.swift]: This function can be static

File: FinancialTracker/ViewControllers/BalanceViewController.swift [hunk 3]
```diff
@@ -15,7 +15,7 @@ class BalanceViewController: UIViewController {
     @IBOutlet var currencyPicker: UIPickerView!
     
     // MARK: - Properties
-    let currency = Currencies()
+    var currencies: [Currency] = []
```
Comment (3257) [inline FinancialTracker/ViewControllers/BalanceViewController.swift]: private?

File: FinancialTracker/ViewControllers/CurrencyTableViewController.swift [hunk 4]
```diff
@@ -18,6 +18,22 @@ class CurrencyTableViewController: UITableViewController {
         self.navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Sign Out", style: .plain, target: self, action: #selector(signOut))
         
         self.tabBarItem.image = UIImage(systemName: "dollarsign.circle")
+        
+        getCurrencies { error, data in
```
Comment (3257) [inline FinancialTracker/ViewControllers/CurrencyTableViewController.swift]: is `data` a suitable name?

**Review by 3257 — CHANGES_REQUESTED** [review]

File: FinancialTracker/Helpers/Currency.swift (hunk 1 above)
Comment (BoyanPavlov03) [reply FinancialTracker/Helpers/Currency.swift]: The property `allCurrencies` is returned in the completion handler.

File: FinancialTracker/Helpers/Currency.swift (hunk 2 above)
Comment (BoyanPavlov03) [reply FinancialTracker/Helpers/Currency.swift]: It does not belong to a class so it is static by default.

File: FinancialTracker/Helpers/Currency.swift (hunk 1 above)
Comment (3257) [reply FinancialTracker/Helpers/Currency.swift]: got it

File: FinancialTracker/Helpers/Currency.swift (hunk 2 above)
Comment (3257) [reply FinancialTracker/Helpers/Currency.swift]: I did not notice. Make it belong to `Currency`

**Review by 3257 — CHANGES_REQUESTED** [review]

**Pull request merged**